### PR TITLE
Dev/ignition 8.1 1.2

### PR DESCRIPTION
**Pull request by alexjhawk**
This PR is mostly just lingering cleanup that I wanted to circle back to.

### CODE REVIEW

File: eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/SyncManager.java [hunk 1]
```diff
@@ -341,12 +341,12 @@ protected void updateLive() {
...
   private HashMap<String, ArrayList<String>> fetchRealtimeTags() {
     // Create hashmap to store Ewons and their realtime tags
-    HashMap<String, ArrayList<String>> liveEwonNames = new HashMap<String, ArrayList<String>>();
+    HashMap<String, ArrayList<String>> liveEwonNames = new HashMap<>();
 
     // Create boolean and string realtime properties
     BasicProperty<Boolean> realtimePropBoolean =
-        new BasicProperty<Boolean>("Realtime", boolean.class);
-    BasicProperty<String> realtimePropString = new BasicProperty<String>("Realtime", String.class);
+            new BasicProperty<>("Realtime", boolean.class);
```
Comment (TomKimsey) [inline eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/SyncManager.java:348]: whitespace change?

File: eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/SyncManager.java [hunk 2]
```diff
@@ -859,7 +859,7 @@ public QualityCode write(TagPath tagPath, Object o) {
               BasicBoundPropertySet readtimeProperty =
                   new BasicBoundPropertySet(
                       new PropertyValue(
-                          new BasicProperty<Boolean>("Realtime", boolean.class), "false"));
+                              new BasicProperty<>("Realtime", boolean.class), "false"));
```
Comment (TomKimsey) [inline eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/SyncManager.java:862]: whitespace change?

**Review by TomKimsey — CHANGES_REQUESTED** [review]

File: eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/SyncManager.java (hunk 2 above)
Comment (alexjhawk) [reply eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/SyncManager.java]: fixed

File: eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/SyncManager.java (hunk 1 above)
Comment (alexjhawk) [reply eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/SyncManager.java]: fixed

**Review by TomKimsey — APPROVED** [review]

**Review by oliver-walker-hms — APPROVED** [review]

**Pull request merged**